src/server/services/schemas/statusSchemaV2.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod'
+
+export const ResponseStatusSchema = z.enum(['NO_RESPONSE', 'TRENGER_OPPFOLGING', 'TRENGER_IKKE_OPPFOLGING'])
+
+export const SenOppfolgingStatusDTOV2Schema = z.object({
```

Comment:
Nitpick: kunne `DTO` vært suffixen så det blir: `SenOppfolgingStatusV2DTOSchema`. I mitt hode er DTO bare en annotasjon for noe.

Reply:
Ja, bør i så fall endre backend også for å ha det i synk :)